src/main/java/com/github/tonivade/resp/protocol/RedisSerializer.java

```diff
@@ -26,76 +26,49 @@
 
   private static final byte[] DELIMITER = new byte[] { '\r', '\n' };
 
-  private ByteBufferBuilder builder = new ByteBufferBuilder();
-
-  public byte[] encodeToken(RedisToken msg) {
-    msg.accept(new AbstractRedisTokenVisitor<Void>() {
-      @Override
-      public Void string(StringRedisToken token) {
-        addBulkStr(token.getValue());
-        return null;
+  public static byte[] encodeToken(RedisToken msg) {
+    ByteBufferBuilder builder = new ByteBufferBuilder();
```

Comment:
I think it can't optimize the problem. It still create ByteBufferBuilder, I think the best way is reuse ByteBufferBuilder.
maybe we can refer `io.netty.util.internal.ObjectPool`.

Reply:
I push another commit using netty [Recycler](https://netty.io/4.1/api/io/netty/util/Recycler.html)